src/imputors/nocb.jl

```diff
@@ -30,13 +24,18 @@ julia> M = [1.0 2.0 missing missing 5.0; 1.1 2.2 3.3 missing 5.5]
  1.0  2.0   missing  missing  5.0
  1.1  2.2  3.3       missing  5.5
 
-julia> impute(M, NOCB(; vardim=1, context=Context(; limit=1.0)))
+julia> impute(M, NOCB(; context=Context(; limit=1.0)); dims=2)
 2×5 Array{Union{Missing, Float64},2}:
  1.0  2.0  5.0  5.0  5.0
  1.1  2.2  3.3  5.5  5.5
 ```
 """
-NOCB(; vardim=2, context=Context()) = NOCB(vardim, context)
+struct NOCB <: Imputor
+    context::AbstractContext
+end
+
+# TODO: Switch to using Base.@kwdef on 1.1
+NOCB(; context=Context()) = NOCB(context)
```

Comment:
Maybe implement the 1.1 version using a version check?

Reply:
Having these constructors is less code than copying that macro in. https://github.com/JuliaLang/julia/blob/c86700ddead1cfa22b30b0eef0012e02bb24d9e7/base/util.jl#L706